jtransc-rt/resources/hx/JA_L.hx

```diff
@@ -15,13 +15,20 @@ class JA_L extends JA_0 {
     }
 
 	{{ HAXE_METHOD_ANNOTATIONS }}
-    static public function fromArray(items:Array<Dynamic>, desc:String):JA_L {
+    static public function fromArray(desc:String, items:Array<Dynamic>):JA_L {
```

Comment:
After some thinking I found that I wanted to be consisent with Tx methods. I prefer to keep variadic part to the end. Thats why I changed the order.
My original thinking was that desc is less relevant and could be even optional so i put it after desc and was an addition to analog methods in other primitive array types. But in the end, an object array need it to be correct.
Also probably i will have to change that string into a proper Type object later to be able to implement instanceof correctly for arrays without tol much overhead. I have to think this good. But probably signature will change again before 1.0.